### PR TITLE
Add markdown-lint.yml

### LINKED ISSUE #7: Setup Github actions and figure out a standard before merging
@wlsf82 I'd like to discuss what are the items we need to ensure if they are working well before pushing to master branch regarding to documentation repository. Any concerns?

### PR DESCRIPTION
**Pull request by pedrohyvo**
Set up github action using markdown lint.

https://github.com/marketplace/actions/markdown-linting-action

Close #7

### CODE REVIEW

File: .github/workflows/markdown-lint.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# This is a basic workflow to help you get started with Actions
```
Comment (wlsf82) [inline .github/workflows/markdown-lint.yml:1]: I'd remove this and the below line.

File: .github/workflows/markdown-lint.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+name: markdown-lint
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (wlsf82) [inline .github/workflows/markdown-lint.yml:11]: I don't think we need this line. I don't use it on protractor-helper, for instance (see .https://github.com/wlsf82/protractor-helper/blob/master/.github/workflows/continuous-integration.yml#L6)

File: .github/workflows/markdown-lint.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```
Comment (wlsf82) [inline .github/workflows/markdown-lint.yml:14]: I'd remove this since we're not using it.

File: .github/workflows/markdown-lint.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  lint-changelog:
+    name: Lint changelog file
```
Comment (wlsf82) [inline .github/workflows/markdown-lint.yml:19]: What about this?

```suggestion
    name: Lint docs
```

File: .github/workflows/markdown-lint.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Lint changelog file
```
Comment (wlsf82) [inline .github/workflows/markdown-lint.yml:25]: Which changelog file?

File: .github/workflows/markdown-lint.yml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+      uses: docker://avtodev/markdown-lint:v1 # fastest way
+      with:
+        rules: '/lint/rules/changelog.js'
+        config: '/lint/config/changelog.yml'
+        args: './CHANGELOG.md'
+        ignore: './one_file.md ./another_file.md' # multiple files must be separated with single space
+
+    # Or using current repository as action:
+
+    - name: Lint changelog file
```
Comment (wlsf82) [inline .github/workflows/markdown-lint.yml:35]: It's the above one or this one, not both.

File: .github/workflows/markdown-lint.yml (hunk 1 above)
Comment (pedrohyvo) [reply .github/workflows/markdown-lint.yml:1]: done

File: .github/workflows/markdown-lint.yml (hunk 2 above)
Comment (pedrohyvo) [reply .github/workflows/markdown-lint.yml:11]: done

File: .github/workflows/markdown-lint.yml (hunk 3 above)
Comment (pedrohyvo) [reply .github/workflows/markdown-lint.yml:14]: done

**Review by wlsf82 — CHANGES_REQUESTED** [review]
@pedrohyvo, I've left a few comments for you.

It seems that you just used an example. We need to adjust it to our needs.

I suggest reading https://medium.com/@christinavhastenrath/github-actions-markdown-lint-setup-86d7f6ec934d.

I don't see any job running the markdown linting on the markdown files.
We should have something like:

```yml
- name: lint all markdownfiles
    run: |
      npm install
      node_modules/.bin/markdownlint ./ --ignore node_modules
```

**Comment by pedrohyvo** [comment]
@wlsf82 now the `markdown lint` is working as expected, but I think we need to create new issues to fix the current docs. Also, I really understand if we need to create this file `mdlint.sh`, that's why I didn't add it into the repo. Let me know your concerns please

**Review by lucastfa — APPROVED** [review]

**Review by wlsf82 — CHANGES_REQUESTED** [review]
Let's fix the linting issues before merging it.

**Pull request merged**